plugins/index.mjs

```diff
@@ -0,0 +1,44 @@
+import path from 'node:path';
+import {fileURLToPath} from 'node:url';
+
+import {BasePlugin} from '@appium/base-plugin';
```

Comment:
We need to change this to 

`import {BasePlugin} from 'appium/plugin.js';`

cc: @mykola-mokhnach

Reply:
Everything LGTM.

Reply:
Done with the latest.